benchmark_runner/workloads/winstress_vm.py

```
@@ -0,0 +1,221 @@
+
+    def _prepare_vm(self, vm_num: str):
+        try:
+            vm_name = self._get_vm_name(vm_num)
+            self._virtctl.start_vm_sync(vm_name=vm_name)
+            if not self._virtctl._wait_for_virtctl_ssh(vm_name=vm_name, namespace=self.__namespace,
+                                                       key_path=self.__ssh_key_path, username=self.__username,
+                                                       timeout=OC.SHORT_TIMEOUT):
+                logger.warning(f'SSH never became ready on VM {vm_name}, skipping')
+                return
+
+        except Exception as err:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)